packages/edit-site/src/components/editor/style.scss

```diff
@@ -26,28 +26,7 @@
 	background-color: $white;
 }
 
-.edit-site-editor__inserter-panel {
+.interface-interface-skeleton__left-sidebar .components-navigation {
+	width: 300px;
 	height: 100%;
```

Comment:
This is probably not the right place for these styles.
What about:
- Remove everything from here.
- Create the `.edit-site-navigation-panel` class as suggested in [another comment](https://github.com/WordPress/gutenberg/pull/25506/files#r493329611).
- Add something like this in `packages/edit-site/src/components/left-sidebar/navigation-panel/style.scss`? 
```scss
.edit-site-navigation-panel {
	height: 100%;
	position: relative;
	width: 300px;

	.components-navigation {
		height: 100%;
	}
}
```